src/FacilitatorOptionParser.ts

```diff
@@ -0,0 +1,179 @@
+import { Config, FacilitatorConfig } from './Config';
+import MosaicConfig from './MosaicConfig';
+import { FacilitatorStartException } from './Exception';
+
+/**
+ * It parses and evaluates input params of facilitator start command to get config object.
+ */
+export default class FacilitatorOptionParser {
+  public originChain: string;
+
+  public auxChain: string;
+
+  public mosaicConfigPath: string;
+
+  public facilitatorConfigPath: string;
+
+  /**
+   * Constructor
+   * @param originChain Name of the origin chain.
+   * @param auxChain Identifier of the aux chain.
+   * @param mosaicConfigPath Path to mosaic config file.
+   * @param facilitatorConfigPath Path to facilitator config path.
+   */
+  public constructor(
+    originChain: string,
```

Comment:
It will always be some string or blank string.
Consider the value of `originChain` is undefined, in the constructor, we define it to `''`(blank string)